packages/extension-browser/package.json

```diff
@@ -99,10 +100,12 @@
     "build:assets": "cpx \"./{src,tests}/**/{!(*.ts),.!(ts)}\" dist && npm-run-all prebuild:*",
     "build:ts": "tsc -b",
     "build:webpack": "webpack --env.content-script && webpack && cpx \"./src/**/*.{html,json,png,svg}\" dist/bundle",
+    "build:web-ext": "web-ext build -s ./dist/bundle -a ./dist --overwrite-dest",
```

Comment:
Should we change this to `build-release`? Or add it as a step of `build-release`?
Ideally we should automate the release process of the extensions as well.

Reply:
Yeah, agree that sounds best.